api_tests/nodes/filters/test_filters.py

```diff
@@ -142,6 +142,13 @@ def test_non_mutating_list_filtering_tests(
         assert child_node_two._id not in ids
         assert project._id in ids
 
+    #   test_root_ne_with_title_id_excludes_children_with_query_in_title_or_id
```

Comment:
So to verify, this gets all nodes that have a title or id equal to the `child_node_two` id as long as they aren't a child of the `root` node, yes?

Reply:
Correct!!